plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/MySqlQueryRunner.java

```diff
@@ -48,9 +49,23 @@ public static DistributedQueryRunner createMySqlQueryRunner(
             Map<String, String> connectorProperties,
             Iterable<TpchTable<?>> tables)
             throws Exception
+    {
+        return createMySqlQueryRunner(server, extraProperties, ImmutableMap.of(), connectorProperties, tables, runner -> {});
```

Comment:
changes to `MySqlQueryRunner` can be moved to `pre-refactoring` commit

Reply:
You mean you want these (and the other pre-refactoring) changes in a separate commit previous to the rest of this PR?

Reply:
same pr - just extra commit